content/concepts/introduction/users.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Who uses libp2p"
+description: libp2p is used by many blockchain networks, p2p applications, and edge computing projects.
+weight: 19
+aliases:
+  - "/introduction/libp2p-users"
+---
+
+When learning about libp2p, it can be helpful to understand how various projects and applications have integrated and benefitted from libp2p over the years.
+As a modular peer-to-peer networking framework, usage of libp2p can look very different when powering large-scale blockchain networks, vs enabling robust p2p applications, vs underpinning flexible edge computing and agent coordination.
+
+A few notable users of libp2p include:
+
+| Project | Description |
+|---------------|---------------------------------|
+| **IPFS** | IPFS initially created and still uses libp2p for decentralized peer-to-peer communication and content distribution. IPFS has achieved significant growth to hundreds of thousands of nodes and billions of weekly content requests, where libp2p enables efficient and secure peer-to-peer communication & interop. |
+| **Filecoin** | Filecoin has become the largest decentralized storage network, surpassing competitors in storage capacity and adoption. It uses libp2p to ensure robust networking, peer discovery, routing, and message propagation - enabling efficient data storage and retrieval. |
+| **Ethereum** | The integration of libp2p into Ethereum 2.0 has been instrumental in achieving scalability and decentralization. It also played a key role in the network's transition to proof-of-stake, which reduced energy consumption by 99.95%. |
+| **Optimism** | Optimism uses libp2p as a core component of its networking infrastructure, particularly in its op-node implementation. |
+| **Celestia** | Celestia uses libp2p for its data availability sampling (DAS) network, which enables light nodes to perform DAS by querying random data shares and verifying their availability. |
+| **Polkadot** | Polkadot leverages libp2p as part of its Substrate-based architecture. Libp2p enables Polkadot's scalability by facilitating seamless communication across its heterogeneous blockchain ecosystem, including parachains and parathreads. |
+| **MetaMask** | MetaMask leverages libp2p's capabilities for cryptographic key management directly in the browser, enhancing security and user control. In addition, MetaMask has created js-libp2p-multicast-conditional, a tool for conditional multicast using libp2p, further expanding the ecosystem's capabilities. |
```

Comment:
Metamask doesn't use libp2p. They had an experiment that never materialised into a long running project.

Reply:
Thanks! clarified.
```suggestion
| **MetaMask** | MetaMask created js-libp2p-multicast-conditional, an experimental tool for conditional multicast using libp2p. |
```